docs/ref/models/indexes.txt

```diff
@@ -92,3 +92,21 @@ opclasses=['jsonb_path_ops'])`` creates a gin index on ``jsonfield`` using
 ``opclasses`` are ignored for databases besides PostgreSQL.
 
 :attr:`Index.name` is required when using ``opclasses``.
+
+``conditions``
```

Comment:
Should it be `condition` instead? The plural form makes me feel like the API is expecting a collection of `Q()`.

Reply:
Yes, I changed it to the singular form.